x-pack/plugins/security_solution/common/endpoint/schema/trusted_apps.ts

```diff
@@ -13,6 +14,12 @@ const hashLengths: readonly number[] = [
 ];
 const hasInvalidCharacters = /[^0-9a-f]/i;
 
+const entryFieldLabels: { [k in TrustedApp['entries'][0]['field']]: string } = {
+  'process.hash.*': 'Hash',
+  'process.executable.caseless': 'Path',
+  'process.code_signature': 'Signer',
```

Comment:
Signer?

Reply:
yeah, its already part of the Type (`TrustedApp`), thus its coming up as required in this object (typescript error). I did not want to exclude it out of the type here in order to keep the type simpler.